generic.xml

```diff
@@ -15,174 +15,2263 @@
+				<param name="elev_geoid" label="elev.geoid" optional="false" help="Use a geoid grid to get the height above the ellipsoid in case there is no DEM available, no coverage for some points or pixels with no_data in the DEM tiles. A version of the geoid can be found on the OTB website(https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb-data/blob/master/Input/DEM/egm96.grd)." type="data" format="txt"/>
+				<param name="elev_default" label="elev.default" optional="false" value="0" help="This parameter allows setting the default height above ellipsoid when there is no DEM available, no coverage for some points or pixels with no_data in the DEM tiles, and no geoid file has been set. This is also used by some application as an average elevation value." type="float"/>
+				<param name="norescale" label="norescale" optional="false" value="False" help="By default, the input image amplitude is rescaled between [0,255]. Turn on this parameter to skip rescaling" type="boolean" truevalue="True" falsevalue="False"/>
+				<param name="ram" label="ram" optional="true" value="128" help="Available memory for processing (in MB)" type="integer"/>
+				<param type="select" name="out_outformat" label="out_outformat" help="Output format">
+					<option value="text/xml">text/xml</option>
+					<option value="application/vnd.google-earth.kml+xml">application/vnd.google-earth.kml+xml</option>
+					<option value="application/json">application/json</option>
+					<option value="application/zip">application/zip</option>
+				</param>
+			</when>
+			<when value="OTB.LocalStatisticExtraction">
+				<param name="in" label="in" optional="false" help="The input image to compute the features on." type="data" format="txt"/>
+				<param name="channel" label="channel" optional="false" value="1" help="The selected channel index" type="integer"/>
+				<param name="ram" label="ram" optional="true" value="128" help="Available memory for processing (in MB)" type="integer"/>
+				<param name="radius" label="radius" optional="false" value="3" help="The computational window radius." type="integer"/>
+				<param name="out" label="out" optional="false" value="float" help="Output image containing the local statistical moments." type="select">
+					<expand macro="out_options"/>
+				</param>
+				<param type="select" name="out_outformat" label="out_outformat" help="Output format">
+					<expand macro="format_options"/>
+				</param>
 			</when>
 		</conditional>
 	</inputs>
 	<outputs>
 		<data name="output_data" format="txt" label="$select_process"/>
 	</outputs>
 	<expand macro="tests"/>
-	<expand macro="help"/>
+	<help>This tool is a wrapper for OGC API Processes (OTB) coming from the Zoo Project (https://zoo-project.github.io/docs/intro.html) and was created using the OGC-API-Process2Galaxy tool (https://github.com/AquaINFRA/OGC-API-Process2Galaxy). Check the README in the repository for more information.</help>
```

Comment:
This should not go into a tool help, please use the `<creator>` tag. The tool help should assist a user in using the tool.